contracts/ERC1155NFT.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract ERC1155NFT is ERC1155, Ownable {
+
+    mapping(uint256 => uint256) private balances;
+
+    mapping(uint256 => address[]) private owners;
+
+    constructor(string memory _uri) ERC1155(_uri) {}
```

Comment:
What about contract Name and Symbol? I know its not required for erc1155 but for example would it show the contract more nicely on Opensea? (Opensea example erc1155 has name and symbol in constructor https://github.com/ProjectOpenSea/opensea-creatures/blob/df8baafd8e8cb60f847b3ede640105f0bad6d057/contracts/ERC1155Tradable.sol)

Reply:
We can put `name` and `symbol` as public variable in our contract constructor, But can you tell me where exactly i can see `name` and `symbol` on Opensea? 

They have default name and symbol for all the contract they deployed : [OpenSea Collections (OPENSTORE)](https://polygonscan.com/address/0x2953399124f0cbb46d2cbacd8a89cf0599974963) & [OpenSea Shared Storefront (OPENSTORE)](https://etherscan.io/address/0x495f947276749ce646f68ac8c248420045cb7b5e)

Reply:
Just tested this contract : https://github.com/ProjectOpenSea/opensea-creatures/blob/df8baafd8e8cb60f847b3ede640105f0bad6d057/contracts/ERC1155Tradable.sol

Opensea by default taking Token name as collection name. 

Opensea Collection : https://testnets.opensea.io/assets/mumbai/0x6f93b31844f45be31c285edc216bdd29f4712c6a/0

Here, token name : BAT TOKENS, rest description is from metadata

Reply:
Just tested `name` and `symbol` in ERC1155, its seems working fine with opensea after token minting. so ya we can go ahead with the same.

Reply:
Solved